extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/resources/translations/es.json

```diff
@@ -0,0 +1,105 @@
+{
+
+    "LOGIN" : {
+
+        "ERROR_PASSWORD_BLANK"    : "@:APP.ERROR_PASSWORD_BLANK",
+        "ERROR_PASSWORD_SAME"     : "La nueva contraseña debe ser diferente a la expirada.",
+        "ERROR_PASSWORD_MISMATCH" : "@:APP.ERROR_PASSWORD_MISMATCH",
+        "ERROR_NOT_VALID"         : "Esta cuenta de usuario no es válida actualmente.",
+        "ERROR_NOT_ACCESSIBLE"    : "Actualmente no se permite el acceso a esta cuenta. Por favor intente de nuevo mas tarde.",
+
+        "INFO_PASSWORD_EXPIRED" : "Su contraseña ha expirado y debe renovarla. Por favor introduzca una nueva contraseña para continuar.",
+
+        "FIELD_HEADER_NEW_PASSWORD"         : "Nueva contraseña",
+        "FIELD_HEADER_CONFIRM_NEW_PASSWORD" : "Confirmar nueva contraseña"
+
+    },
+
+    "CONNECTION_ATTRIBUTES" : {
+
+        "FIELD_HEADER_MAX_CONNECTIONS"          : "Número máximo de conexiones:",
+        "FIELD_HEADER_MAX_CONNECTIONS_PER_USER" : "Número máximo de conexiones por usuario:",
+
+        "FIELD_HEADER_FAILOVER_ONLY"            : "Usar solo para failover:",
+        "FIELD_HEADER_WEIGHT"                   : "Peso de la conexión:",
+
+        "FIELD_HEADER_GUACD_HOSTNAME"   : "Nombre de Host:",
+        "FIELD_HEADER_GUACD_ENCRYPTION" : "Encriptación:",
+        "FIELD_HEADER_GUACD_PORT"       : "Puerto:",
+
+        "FIELD_OPTION_GUACD_ENCRYPTION_EMPTY" : "",
+        "FIELD_OPTION_GUACD_ENCRYPTION_NONE"  : "Ninguna (sin encriptar)",
+        "FIELD_OPTION_GUACD_ENCRYPTION_SSL"   : "SSL / TLS",
+
+        "SECTION_HEADER_CONCURRENCY"    : "Límites de concurrencia",
+        "SECTION_HEADER_LOAD_BALANCING" : "Balanceo de carga",
+        "SECTION_HEADER_GUACD"          : "Parámetros de Proxy Guacamole (guacd)"
+
+    },
+
+    "CONNECTION_GROUP_ATTRIBUTES" : {
+
+        "FIELD_HEADER_ENABLE_SESSION_AFFINITY"  : "Enable session affinity:",
```

Comment:
Not to be greedy, but what about the other remaining English? Or are these not readily translatable?